cmd/pulumi.go

```diff
@@ -76,6 +82,24 @@ func NewPulumiCmd() *cobra.Command {
 				}
 			}
 
+			if rsp, err := http.Get("https://pulumi.io/latest-version"); err == nil && rsp.StatusCode == http.StatusOK {
+				buf := bytes.Buffer{}
+				buf.ReadFrom(rsp.Body)
+
+				latestVer, latestErr := semver.ParseTolerant(buf.String())
+				curVer, curErr := semver.ParseTolerant(version.Version)
+
+				if latestErr == nil && curErr == nil {
+					if latestErr == nil && curErr == nil && !isDevVersion(curVer) {
```

Comment:
Why repeat `latestErr == nil && curErr == nil`?

Reply:
Didn't delete this extra line while refactoring.